### PR TITLE
Fix admin test commands bug

### PR DESCRIPTION
**Pull request by DanieloV**
Small bug fix for the test commands. 
Currently an admin generates runtime errors when writing short texts in the chat, even if they are not test commands.
Also takes care of a few other corner cases where errors can occur.

### CODE REVIEW

File: game/match/match.go [hunk 1]
```diff
@@ -1801,7 +1801,7 @@ func (m *Match) handleAdminMesseges(message string, user db.User) {
 	// /mana red 4 - add 4 fire mana
 	// /mana n - add 1 nature mana
 	// /mana imageID 3 - add 3 mana of that specific card
-	if string(message[0:5]) == "/mana" {
+	if string(message[0:6]) == "/mana " {
 
 		msgParts := strings.Split(message, " ")
```
Comment (sindreslungaard) [inline game/match/match.go:1806]: This could probably be moved outside of the individual commands and used to test for the first word so that we don't need to have the `len(message) < 7` check above but instead check that it's at least 1 word long in case someone adds a new command with more than 7 characters in the future without updating the length check

Comment (DanieloV) [reply game/match/match.go]: ok, changed it
